components/x-teaser/src/v2/components/Title/styles.scss

```diff
@@ -0,0 +1,56 @@
+@import '@financial-times/o-typography/main';
+@import '@financial-times/o-colors/main';
+@import '@financial-times/o-labels/main';
+@import '../../themes/themes';
+
+.title {
+	// @TODO: this needs a proper implementation via origami's o-fonts/o-fonts-assets;
+	font-family: FinancierMedium, $o-typography-display, sans-serif;
+}
+
+.titleLink {
+	@include themify {
+		color: themed('titleTextColor');
+
+		&:hover,
+		&:focus {
+			color: themed('titleTextColorHover');
+		}
+	}
+
+	text-decoration: none;
+	vertical-align: middle;
+	margin-right: 5px;
```

Comment:
Is the `5px` arbitrary? Instead of set `px` values it would be great to use [o-spacing](https://github.com/Financial-Times/o-spacing) or a relative unit like `rem`, `ch`, `ex` -- makes things a little more consistent and amenable to changes. If it makes sense here

Reply:
Checked with Mark and 4px is fine, so switched to `s1` spacing